### PR TITLE
Update edit session modal

### PR DESCRIPTION
**Pull request by transient-jonas**

### CODE REVIEW

File: shared/src/schemas/Session.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ const LiveSessionFieldsSchema = yup
     url: yup.string().required(),
     link: yup.string(),
     inviteCode: yup.number().required(),
+    hostingCode: yup.number(),
```
Comment (gewfy) [inline shared/src/schemas/Session.ts:43]: Before merging this make sure this is not included in any session responses

**Review by gewfy — COMMENTED** [review]
Know it's WIP but just skimmed through it out of curiosity :) Looks good! 🙌

File: shared/src/schemas/Session.ts (hunk 1 above)
Comment (transient-jonas) [reply shared/src/schemas/Session.ts:43]: Where does the field selection/exclusion happens?
It is taking some time to be able to find it in the code, doesn't seem to be in neither the modal, controller or route 🤔

**Review by swemail — APPROVED** [review]
Nice work!

**Pull request merged**